docs/diagrams/DeleteSingleIndexSequenceDiagram.puml

```diff
@@ -1,11 +1,12 @@
 @startuml
 !include style.puml
+hide footbox
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":DengueHotspotTrackerParser" as DengueHotspotTrackerParser LOGIC_COLOR
 participant ":DeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
-participant "d:DeleteCommand" as DeleteCommand LOGIC_COLOR
```

Comment:
![image](https://user-images.githubusercontent.com/100463724/230766460-3f0d5be0-2155-4ead-9e2a-c95fd3cf9be1.png)
ngl from what i see in ab3 i think the d is actually necessary, since you're passing the d around it your other commands!

Reply:
Good catch! Updated, thanks!